README.md

```diff
@@ -177,6 +177,16 @@ You can also join our discord server
   <img src="https://img.shields.io/discord/1064533716615049236.svg?style=for-the-badge" alt="Discord Channel">
 </a>
 
+## Acknowledgments
+
+[Python Software Foundation (PSF)](https://www.python.org/psf/) grants PyMilo library partially for version **1.0**. [PSF](https://www.python.org/psf/) is the organization behind Python. Their mission is to promote, protect, and advance the Python programming language and to support and facilitate the growth of a diverse and international community of Python programmers.
+
+<a href="https://www.python.org/psf/"><img src="https://github.com/sepandhaghighi/pycm/raw/master/Otherfiles/PSF.png" height="55px" alt="Python Software Foundation"></a>
```

Comment:
Are you sure you want to point out to `PSF.ong` in `sepandhaghighi/pycm`?
I would say it's better to add it to repository or point out to their website ([here](https://wiki.python.org/psf/PSF%20Press%20Kit)).

Reply:
My bad! Thank you. 
I added the logo separately.